resources/io/jenkins/infra/docker/pod-template.yml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: "v1"
+kind: "Pod"
+metadata:
+  labels:
+    jenkins: "agent"
```

Comment:
I would be interested for label which could easily identify this job  like `job: docker-builder`

Reply:
I need a precision on this: should the value of the label be static (eg. always `docker-builder`)  or should it be the name of the image currently being built (e.g. `docker-datadog`, `docker-terraform` etc. ?)

Reply:
The goal is to easily and clearly identify logs for a specific  job, so using dynamic names would be even better

Reply:
OK, this will need a bit more work then

Reply:
It's done: here is the final pod from the build logs (https://infra.ci.jenkins.io/job/Docker%20Builds/job/docker-builder/job/PR-10/24/console):

```
---
apiVersion: "v1"
kind: "Pod"
metadata:
  annotations:
    container.apparmor.security.beta.kubernetes.io/builder: "unconfined"
    container.seccomp.security.alpha.kubernetes.io/builder: "unconfined"
    buildUrl: "http://jenkins-infra:8080/job/Docker%20Builds/job/docker-builder/job/PR-10/24/"
    runUrl: "job/Docker%20Builds/job/docker-builder/job/PR-10/24/"
  labels:
    jenkins: "agent"
    job/kind: "docker-image-build"
    docker-image/name: "builder"
    jenkins/jenkins-infra-agent: "true"
    jenkins/label-digest: "168581a5f7dd7bc78734253113adb8b4115e61ba"
    jenkins/label: "Docker_Builds_docker-builder_PR-10_24-tb7k6"
  name: "docker-builds-docker-builder-pr-10-24-tb7k6-zz4t4-tsjxp"
spec:
  containers:
  - command:
    - "cat"
    image: "jenkinsciinfra/builder:latest"
    name: "builder"
    tty: true
    volumeMounts:
    - mountPath: "/home/jenkins/agent"
      name: "workspace-volume"
      readOnly: false
  - args:
    - "********"
    - "docker-builds-docker-builder-pr-10-24-tb7k6-zz4t4-tsjxp"
    env:
    - name: "JENKINS_SECRET"
      value: "********"
    - name: "JENKINS_TUNNEL"
      value: "jenkins-infra-agent:50000"
    - name: "JENKINS_AGENT_NAME"
      value: "docker-builds-docker-builder-pr-10-24-tb7k6-zz4t4-tsjxp"
    - name: "JENKINS_NAME"
      value: "docker-builds-docker-builder-pr-10-24-tb7k6-zz4t4-tsjxp"
    - name: "JENKINS_AGENT_WORKDIR"
      value: "/home/jenkins/agent"
    - name: "JENKINS_URL"
      value: "http://jenkins-infra:8080/"
    image: "jenkins/inbound-agent:latest-jdk11"
    imagePullPolicy: "Always"
    name: "jnlp"
    resources:
      limits:
        memory: "512Mi"
        cpu: "500m"
      requests:
        memory: "512Mi"
        cpu: "500m"
    tty: false
    volumeMounts:
    - mountPath: "/home/jenkins/agent"
      name: "workspace-volume"
      readOnly: false
  nodeSelector:
    kubernetes.io/os: "linux"
  restartPolicy: "Never"
  volumes:
  - emptyDir:
      medium: ""
    name: "workspace-volume"
```

Reply:
It defines 2 labels:

- `job/kind` which is static and always the same value `docker-image-build`
- `docker-image/name` which is dynamic and holds the Docker Image name being built

@olblak is it ok for you?

Reply:
Yeaaa that's awesome, thank about that.